hw/ip/lc_ctrl/data/lc_ctrl.hjson

```diff
@@ -720,6 +750,23 @@
           device is in a non-PROD life cycle state. The clock mux will remain switched until the next system reset.
           '''
         }
+        { bits: "1",
+          name: "VOLATILE_RAW_UNLOCK",
+          desc: '''
+          When set to 1, LC_CTRL performs a volatile lifecycle transition from RAW -> TEST_UNLOCKED0.
+          No state update will be written to OTP, and no reset will be needed after the transition has succeeded.
+          Note that the token to be provided has to be the hashed unlock token, since in this case the token is NOT passed through KMAC before performing the comparison.
+
+          After a successful VOLATILE_RAW_UNLOCK transition from RAW -> TEST_UNLOCKED0, the LC_CTRL FSM will go back to the IdleSt and set the STATUS.TRANSITION_SUCCESSFUL bit.
+          The LC_CTRL accepts further transition commands in this state.
+
+          IMPORTANT NOTE: this feature is intended for test chips only in order to mitigate the risks of a malfunctioning
+          OTP macro. Production devices will permanently disable this feature at compile time via the SecVolatileRawUnlockEn parameter.
+
+          Software can check whether VOLATILE_RAW_UNLOCK is available by writing 1 and reading back
+          the register value. If the register reads back as 1 the mechanism is available, and if it reads back 0 it is not.
```

Comment:
Question regarding strap re-sampling. If the user wants to switch to CPU JTAG after exit, is this the correct set of operations?

1. Select lc_ctrl TAP
2. Verify in lc_ctrl TAP by reading a register (e.g. HW_REV)
3. Modify strap pins to select next TAP interface after successful `VOLATILE_RAW_UNLOCKED`
4. Wait for sometime
5. Check lc_ctrl state through CPU JTAG.

Reply:
you can poll on the `STATUS.TRANSITION_SUCCESSFUL` bit for step 4.

for 5. that would work yes.

Reply:
ack sounds good. Just wanted to check this flow, as in the open source we will probably want to test the OTP front door right after 5 (for reference manuf).